<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupTools</class>
 <widget class="QDialog" name="GroupTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>160</width>
     <height>218</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="GroupTools_2">
    <item>
     <widget class="QPushButton" name="ManagerList">
      <property name="text">
       <string>管理员名单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ChangeManager">
      <property name="text">
       <string>更改管理员</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PeopleGroup">
      <property name="text">
       <string>群成员名单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ChangePeople">
      <property name="text">
       <string>更改群成员</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
